doc/rst/tools/chplcheck/chplcheck.rst

```diff
@@ -266,6 +295,74 @@ This function performs _two_ pattern-based searches: one for formals, and one
 for identifiers that might reference the formals. It then emits a warning for
 each formal for which there wasn't a corresponding identifier.
 
+Making Rules Ignorable
+~~~~~~~~~~~~~~~~~~~~~~
+
+If a lint rule is ignorable, it should be marked as such. Ignorable rules
+automatically have a fixit associated with them which adds a
+``@chplcheck.ignore`` annotation to the node.
+
+Ignorable basic rules should return ``BasicRuleResult`` with ``ignorable`` set
+to ``True`` rather than just a boolean. The ``BasicRuleResult`` constructor
+takes a ``AstNode`` as an argument, which is the node that the rule is being
+applied to. For example, the following defines a basic rule that is ignorable:
+
+.. code-block:: python
+
+   @driver.basic_rule(chapel.Function)
+   def NoFunctionFoo(context, node):
+       if node.name() == "foo":
+           return BasicRuleResult(node, ignorable=True)
+       return True
+
+Ignorable advanced rules should yield a ``AdvancedRuleResult`` with ``anchor``
+set rather than just a ``AstNode``. The ``AdvancedRuleResult`` constructor
+takes an ``AstNode`` as an argument, which is the node that the rule is being
+applied to. The ``anchor`` is the node should have a ``@chplcheck.ignore``
+annotation to suppress the warning. ``anchor`` and ``node`` can be the same
+node. For example, the following defines an advanced rule that is ignorable:
+
+.. code-block:: python
+
+   @driver.advanced_rule
+   def NoLoopIndexI(context, root):
+       for loop, _ in chapel.each_matching(root, IndexableLoop):
+           idx = loop.index()
+           if idx.name() == "i":
+               yield AdvancedRuleResult(idx, anchor=loop)
+
+Fixits
+~~~~~~
```

Comment:
Move this above ignorable rules since that section mentions fixits? Or at least link it from above.

Reply:
I made it a link